pkg/driver/node.go

```diff
@@ -117,6 +117,8 @@ func (d *nodeService) NodeStageVolume(ctx context.Context, req *csi.NodeStageVol
 		return &csi.NodeStageVolumeResponse{}, nil
 	}
 
+	fsType := volCap.GetMount().GetFsType()
```

Comment:
Please check for nil from `GetMount()` and empty string from `GetFsType()`

Reply:
done